glui_textbox.cpp

```diff
@@ -180,8 +180,9 @@ int    GLUI_TextBox::key_handler( unsigned char key,int modifiers )
   /*  has_selection = (sel_start != sel_end);              */
 
   if ( key  == CTRL('[')) {         /* ESCAPE */
-    glui->deactivate_current_control();
-    return true;
+    
+	  //GLUI_Main::keyboard returns control to the container.
+	  assert(0); 
```

Comment:
It should, because there used to be code here. And so controls should never receive an Esc key. It's to keep programmers from wasting time in trying to handle the Esc key.